tests/conftest.py

```diff
@@ -139,6 +138,12 @@ def reporter(request, env):
 
 @pytest.fixture(scope='function')
 def os_user(monkeypatch):
+    if os.name == 'nt':
+        import getpass
```

Comment:
I moved that imports to the top of the file, but It was making [tests fail](https://travis-ci.org/lektor/lektor/jobs/253404336), I think that is because It's a fixture, so I deleted that commit